string time
int64 id
string c_mode
int64 num_p
rosbridge/p_state[] p_state
